tools/api-markdown-documenter/src/test/snapshots/html/simple-suite-test/sparse-config/test-suite-a/testfunctionreturninguniontype-function.html

```diff
@@ -1,36 +1,21 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
   <head>
-    <meta charset="utf-8" />
+    <meta charset="utf-8">
   </head>
   <body>
     <section>
-      <h2>
-        testFunctionReturningUnionType
-      </h2>
+      <h2>testFunctionReturningUnionType</h2>
       <section>
-        <p>
-          Test function that returns an inline type
-        </p>
+        <p>Test function that returns an inline type</p>
       </section>
       <section>
-        <h3 id="testfunctionreturninguniontype-signature">
-          Signature
-        </h3>
-        <code>
-          export declare function testFunctionReturningUnionType(): string &#124; TestInterface;
-        </code>
+        <h3 id="testfunctionreturninguniontype-signature">Signature</h3><code>export declare function testFunctionReturningUnionType(): string | TestInterface;</code>
       </section>
       <section>
-        <h3 id="testfunctionreturninguniontype-returns">
-          Returns
-        </h3>
-        <p>
-          A union type
-        </p>
-        <p>
-          <span><b>Return type:</b> </span><span>string &#124; <a href='docs/test-suite-a/testinterface-interface'>TestInterface</a></span>
-        </p>
+        <h3 id="testfunctionreturninguniontype-returns">Returns</h3>
+        <p>A union type</p>
+        <p><span><b>Return type: </b></span><span>string | <a href="docs/test-suite-a/testinterface-interface">TestInterface</a></span></p>
```

Comment:
`&#124; -> |`. Not sure that `|` has any special meaning in HTML syntax so I think it might be ok, just like for the quotes in a previous comment, but also worth checking, since some of the changes of this kind do seem incorrect to me.

Reply:
Yup, this is valid.
![image](https://github.com/user-attachments/assets/c35b3b81-dcf7-4e95-916b-6e3b28bf7b58)